arrabbiata/src/column.rs

```diff
@@ -18,12 +18,19 @@ pub enum Gadget {
     // Elliptic curve related gadgets
     EllipticCurveAddition,
     EllipticCurveScaling,
-    /// This gadget implement the Poseidon hash instance described in the
-    /// top-level documentation. This implementation does use the "next row"
-    /// to allow the computation of one additional round per row. In the current
-    /// setup, with [crate::NUMBER_OF_COLUMNS] columns, we can compute 5 full
+    /// The following gadgets implement the Poseidon hash instance described in
+    /// the top-level documentation. In the current setup, with
+    /// [crate::NUMBER_OF_COLUMNS] columns, we can compute 5 full
     /// rounds per row.
     Poseidon,
```

Comment:
Note: the old Poseidon gadget will be removed later.
By introducing incrementally the gadget `PoseidonPermutation`, I hope it will ease the reviewer work.